cookbooks/bcpc-hadoop/metadata.rb

```diff
@@ -7,6 +7,7 @@
 version          "0.1.0"
 
 depends "bcpc", ">= 0.5.0"
+depends "bach_sqoop"
```

Comment:
I would rather not see` node[:bcpc][:hadoop]` attributes used here. Please use `node[:bach][:sqoop]` and use a wrapper in bcpc-hadoop to set the `node[:bach][:sqoop]` attributes needed (like Hannibal does).